### PR TITLE
Adding cpp example for motivation

### PR DESCRIPTION
**Pull request by PoojanSmart**
Adding a c++ example in `motivation.md`

### CODE REVIEW

File: content/motivation.md [hunk 1]
```diff
@@ -46,7 +46,43 @@ to establish accuracy:
        expected_result = 37.777777
        assert abs(temp_c - expected_result) < 1.0e-6
    ```
-
+   ```{code-tab} c/c++
```
Comment (robertodr) [inline content/motivation.md:49]: You need `c++` as argument, otherwise the Pygments lexer gets confused on how to generate highlighting

**Comment by wikfeldt** [comment]
thanks a lot @PoojanSmart for adding this! Like Roberto points out the `{code-tab} c/c++` directives need to be changed to `{code-tab} c++`. Can you update this in follow-up commits and then we merge?

**Comment by PoojanSmart** [comment]
Sure

**Comment by PoojanSmart** [comment]
Hey, I have added the cpp code. Please review and merge.

**Comment by robertodr** [comment]
Is this PR a duplicate of #121? If yes, please close it.

**Comment by wikfeldt** [comment]
it looks like this is a duplicate of #121, just without the tab directive. So i'll close it

**Pull request closed without merging**